Colour.js

```diff
@@ -7,102 +7,99 @@
  * 3. Convert XYZ to LAB
  * 4. Calculate Delta E00 between two LAB colour (Main purpose)
  * @author Ahmed Moussa <moussa.ahmed95@gmail.com>
- * @version 1.7
+ * @version 2.0
  */
  class Colour {
 	/**
 	 * Convert HEX to LAB
 	 * @param {[string]} hex hex colour value desired to be converted to LAB
 	 */
-	static HEX2LAB(hex) {
-		const [R, G, b, a] = Colour.HEX2RGBA(hex);
-		const [X, Y, Z] = Colour.RGB2XYZ(R, G, b, a);
-		const [L, A, B] = Colour.XYZ2LAB(X, Y, Z);
-		return [L, A, B];
+	static hex2lab(hex) {
+		const [r, g, b, a] = Colour.hex2rgba(hex);
+		const [x, y, z] = Colour.rgb2xyz(r, g, b, a);
+		return Colour.xyz2lab(x, y, z); // [l, a, b]
 	}
 	/**
 	 * Convert RGBA to LAB
-	 * @param {[Number]} R     Red value from 0 to 255
-	 * @param {[Number]} G     Green value from 0 to 255
-	 * @param {[Number]} B     Blue value from 0 to 255
+	 * @param {[Number]} r     Red value from 0 to 255
+	 * @param {[Number]} g     Green value from 0 to 255
+	 * @param {[Number]} b     Blue value from 0 to 255
 	 */
-	static RGBA2LAB(R, G, b, a = 1) {
-		const [X, Y, Z] = Colour.RGB2XYZ(R, G, b, a);
-		const [L, A, B] = Colour.XYZ2LAB(X, Y, Z);
-		return [L, A, B];
+	static rgba2lab(r, g, b, a = 1) {
+		const [x, y, z] = Colour.rgb2xyz(r, g, b, a);
+		return Colour.xyz2lab(x, y, z); // [l, a, b]
 	}
 	/**
 	 * Convert LAB to RGBA
-	 * @param {[Number]} L
-	 * @param {[Number]} A
-	 * @param {[Number]} B 
+	 * @param {[Number]} l
+	 * @param {[Number]} a
+	 * @param {[Number]} b 
 	 */
-	static LAB2RGBA(L, A, B) {
-		const [X, Y, Z] = Colour.LAB2XYZ(L, A, B);
-		const [R, G, b, a] = Colour.XYZ2RGBA(X, Y, Z);
-		return [R, G, b, a];
+	static lab2rgba(l, a, b) {
+		const [x, y, z] = Colour.lab2xyz(l, a, b);
+		return Colour.xyz2rgba(x, y, z); // [r, g, b, a]
 	}
 	/**
 	 * Convert HEX to RGBA
 	 * @param {[string]} hex hex colour value desired to be converted to RGBA
 	 */
-	static HEX2RGBA(hex) {
+	static hex2rgba(hex) {
 		let c;
-		if (hex.charAt(0) == "#") {
+		if (hex.charAt(0) === "#") {
 			c = hex.substring(1).split('');
 		}
 		if (c.length > 6 || c.length < 3) {
 			throw new Error(`HEX colour must be 3 or 6 values. You provided it ${c.length}`);
 		}
-		if (c.length == 3) {
+		if (c.length === 3) {
 			c = [c[0], c[0], c[1], c[1], c[2], c[2]];
 		}
-		c = '0x' + c.join('');
-		let R = (c >> 16) & 255;
-		let G = (c >> 8) & 255;
-		let B = c & 255;
-		let A = 1;
-		return [R, G, B, A];
+		c = "0x" + c.join('');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/hamada147/IsThisColourSimilar/pullRequest?prid=7697707)
```suggestion
		c = "0x" + c.join("");
```